core/extraction/utils.py

```diff
@@ -109,3 +110,40 @@ def join_as_string(values: Sequence) -> str:
     :return: A string of all input values with ", " separating.
     """
     return ", ".join(str(value) for value in values)
+
+
+def drop_unnecessary_fhsf_data(df: pd.DataFrame):
+    """
+    Drop all unwanted funding data for Future High Street Fund and unwanted dates
```

Comment:
can you add to this docstring a bit about intended use i.e. it's to remove extra unwanted data from R1 / R2 towns fund data? And which dataframe it actually is (Funding).